soroban-sdk/src/lib.rs

```diff
@@ -767,6 +767,7 @@ pub mod iter;
 pub mod ledger;
 pub mod logs;
 mod map;
+pub mod prng;
```

Comment:
@graydon Do you feel strongly about this package and type being named "prng"? I think rnd instead of rng is more common abbreviation in libs and APIs and may be more easily recognizable. I don't feel strongly if this is important as is.